src/KafkaFlow.IntegrationTests/Core/Middlewares/GzipMiddleware.cs

```diff
@@ -1,13 +1,17 @@
 namespace KafkaFlow.IntegrationTests.Core.Middlewares
 {
+    using System.Diagnostics;
     using System.Threading.Tasks;
     using KafkaFlow.IntegrationTests.Core.Handlers;
 
     internal class GzipMiddleware : IMessageMiddleware
     {
         public async Task Invoke(IMessageContext context, MiddlewareDelegate next)
         {
-            MessageStorage.Add((byte[]) context.Message.Value);
+            var source = new ActivitySource("KafkaFlow.OpenTelemetry");
```

Comment:
Please rebase with master since this has changed in the other PR https://github.com/Farfetch/kafkaflow/pull/476/

Reply:
Done, please review